### ISSUE TITLE
Remove useless PHPDoc tags

### ISSUE THREAD

**Issue by ntzm** [FriendsOfPHP/PHP-CS-Fixer#3690]
As suggested by https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3651#issuecomment-377224053

```diff
 /**
  * Does foo.
  *
  * @param string $bar Has a description so shouldn't be removed
- * @param mixed $baz
- *
- * @return array
  */
 function foo(string $bar, $baz): array
 {
 }
```

**Comment by Slamdunk** [comment]
Little note: run this after #3640

**Comment by julienfalque** [comment]
Currently covered by #3110.

**Comment by julienfalque** [comment]
This is actually a duplicate of #2979.
